docs/card/index.html

```diff
@@ -0,0 +1,203 @@
+          <a class="sidebar-btn" href="/docs/image/index.html">Image</a>
+          <a class="sidebar-btn" href="/docs/input/index.html">Input</a>
+          <a class="sidebar-btn" href="/docs/text/index.html">Text Utilities</a>
+          <a class="sidebar-btn" href="/docs/list/index.html">Lists</a>
+          <a class="sidebar-btn" href="/docs/navigation/index.html">Navigation</a>
+          <a class="sidebar-btn" href="/docs/toast/index.html">Toast</a>
+          <a class="sidebar-btn" href="/docs/rating/index.html">Rating</a>
+          <a class="sidebar-btn" href="/docs/grid/index.html">Simplified Grid</a>
+          <a class="sidebar-btn" href="/docs/modal/index.html">Modal</a>
+        </div>
+      </div>
+      <div class="content">
+        <h1 class="cl-heading">Cards</h1>
+        <p class="cl-description">
+            A card is a flexible and extensible content container. It includes options for headers and footers, a wide variety of content, contextual background colors, and powerful display options.
+        </p>
+        <h2 class="cl-sub-heading">Text Only card</h2>
+        <p class="cl-text">This card can be used for displaying text only.</p>
+        <div class="showcase showcase-col" >
+          <!-- Text only cards -->
+          <div class="card">
+            <p class="card-title">Our changing planet</p>
+            <p class="card-subtitle">By Kurl Wagner</p>
+            <p class="card-description">
+              Visit ten places on our planet that are under going the biggest
+              changes today.
+            </p>
+          </div>
+        </div>
+
+        <iframe
+          src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=material&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card%2522%253E%250A%2520%2520%253Cp%2520class%253D%2522card-title%2522%253EOur%2520changing%2520planet%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522card-subtitle%2522%253EBy%2520Kurl%2520Wagner%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522card-description%2522%253E%250A%2520%2520%2520%2520Visit%2520ten%2520places%2520on%2520our%2520planet%2520that%2520are%2520under%2520going%2520the%2520biggest%2520changes%2520today.%250A%2520%2520%253C%252Fp%253E%250A%253C%252Fdiv%253E"
+          style="width: 680px; height: 318px; border:0; transform: scale(1); overflow:hidden;"
+          sandbox="allow-scripts allow-same-origin">
+        </iframe>
+
+        <h2 class="cl-sub-heading">Card with Badges</h2>
+        <p class="cl-text">Cards can be paired with Badges as well, to learn more about badge look into the badges section.</p>
+        <div class="showcase showcase-col" >
+          <!-- Cards with badges -->
+          <div class="card relative" >
+            <img class="card-image" src="https://picsum.photos/200/150" alt="" />
```

Comment:
Please provide a meaningful alt description

Reply:
ok got it.